package.json

```diff
@@ -1,36 +1,35 @@
 {
   "name": "gpii-pouchdb",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "private": false,
   "repository": "https://github.com/GPII/gpii-pouchdb",
   "license": "BSD-3-Clause",
   "scripts": {
     "pretest": "node node_modules/rimraf/bin.js reports/*",
     "test": "npm run test:browser && npm run test:node",
-    "test:browser": "node node_modules/testem/testem.js ci --skip safari",
+    "test:browser": "node node_modules/testem/testem.js ci --skip safari,phantomjs",
     "test:node": "node node_modules/istanbul/lib/cli.js cover tests/js/all-tests.js --print summary"
   },
   "dependencies": {
     "express": "4.14.0",
     "express-pouchdb": "https://github.com/the-t-in-rtf/express-pouchdb.git#fa2778d1fdb6ac7c9d8adf7894b12df2e9e134f0",
-    "gpii-express": "1.0.0",
-    "infusion": "2.0.0-dev.20161027T155641Z.0bcd507",
+    "gpii-express": "1.0.2",
+    "infusion": "2.0.0",
```

Comment:
May as well update to the infusion which is seen in your/our other current projects, which is right now 3.0.0-dev.20170131T153243Z.6aab53a